gaatha/settings.py

```diff
@@ -0,0 +1,175 @@
+"""
+Django settings for gaatha project.
+
+Generated by 'django-admin startproject' using Django 4.1.5.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/4.1/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/4.1/ref/settings/
+"""
+import os
+from pathlib import Path
+import environ
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+env = environ.Env(
+    DEBUG=(bool, True),
+    SECRET_KEY=(str, 'django-insecure-pm5c((7v2kh7btd(o%2exi4+6_ui1pso-6naxq-6zuaxs7sq+%'),
+    ALLOWED_HOSTS=(list, ['*']),
+    DB_NAME=(str, 'postgres'),
+    DB_USER=(str, 'postgres'),
+    DB_PWD=(str, 'postgres'),
+    DB_HOST=(str, 'db'),
+    DB_PORT=(str, 5432),
```

Comment:
Let's not define default values here. Use docker-compose as a stack to define there. From an application point-of-view, the application doesn't have any knowledge of the stack it is running on.
Also, DB_PORT is `int` not `str`.
```suggestion
    DB_NAME=str,
    DB_USER=str,
    DB_PWD=str,
    DB_HOST=str,
    DB_PORT=int,
```